blocks/date-block/package.json

```diff
@@ -26,8 +26,7 @@
   },
   "peerDependencies": {
     "@wpmedia/engine-theme-sdk": "canary",
-    "@wpmedia/news-theme-css": "stable",
-    "styled-components": "^4.4.0"
+    "@wpmedia/news-theme-css": "stable"
   },
```

Comment:
can you add shared-styles as a dependency?

Reply:
yes, thanks for catching that